src/extensions/default/CSSCodeHints/CSSGlobalValues.json

```diff
@@ -0,0 +1,7 @@
+[
+    "inherit",
+    "initial",
+    "revert",
+    "unset",
+    "var()"
```

Comment:
In this case, on inserting hint cursor will be after ")". Which is not much useful. Because user need to place cursor manually.